### PR TITLE
Issue #3204 Luau Comments

### PR DESCRIPTION
**Pull request by AnastasiaCortes**
### Description
Adding a new entry regarding Luau comments

### Type of Change

- Adding a new entry

### Checklist

 - [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@AnastasiaCortes I did a first review and left a couple of comments.

File: content/luau/concepts/comments/comments.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+Title: 'Comments' 
+Description: 'Comments in Luau are used to add explanatory or descriptive notes within the code. They are ignored by the interpreter and do not affect the execution of the program.'
```
Comment (caupolicandiaz) [inline content/luau/concepts/comments/comments.md:3]:
```suggestion
Description: 'Comments are used to add explanatory or descriptive notes within a script or program.'
```

File: content/luau/concepts/comments/comments.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  - 'Game Development'
+Tags: 
+  - 'Comments'
+  - 'Documentation'
+CatalogContent: 
+  - 'learn-luau'
+  - 'paths/computer-science'
+---
+
+A **comment**  is text that the Luau parser ignores at runtime. They are useful for documenting code, leaving reminders for future modifications, or temporarily disabling code without removing it.
```
Comment (caupolicandiaz) [inline content/luau/concepts/comments/comments.md:15]:
```suggestion
A **comment** is text that the Luau parser ignores at runtime. They are useful for documenting code, leaving reminders for future modifications, or temporarily disabling code without removing it.
```

File: content/luau/concepts/comments/comments.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+## Single-line Comments
+
+In Luau, single-line comments are created witha double hyphen `--`. Single-line comments extend to the end of the line.
+
+```lua
+type Point = { x: number, y :number }
```
Comment (caupolicandiaz) [inline content/luau/concepts/comments/comments.md:22]:
```suggestion
type Point = { x: number, y: number }
```

File: content/luau/concepts/comments/comments.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+-- This is a single-line comment.
+-- It will not affect the execution of the program.
+local p: Point = { x = 3, y = 4 }
+print(p.x, p.y)
+```
+
+## Multi-line Comments
+
+In Luau, multi-line or block comments are created using double hyphens and double brackets `--[[ ]]`. The compiler does not read any text in between. Multi-line comments commonly used at the top of files to describe their purpose or before functions to describe their intent.
```
Comment (caupolicandiaz) [inline content/luau/concepts/comments/comments.md:31]:
```suggestion
In Luau, multi-line or block comments are created using double hyphens and double brackets `--[[ ]]`. The compiler does not read any text in between. Multi-line comments are commonly used at the top of files to describe the purpose of a library or module, before functions to describe their intent, and in many other cases.
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@AnastasiaCortes I've completed my initial review. Please update the entry, and tag me when you've finished making changes.

**Comment by AnastasiaCortes** [comment]
@caupolicandiaz finished with the changes.

**Review by caupolicandiaz — APPROVED** [review]
@AnastasiaCortes @SSwiniarski Looks good to merge

**Pull request merged**